### PR TITLE
Add descriptions to categories on hover

### PR DESCRIPTION
**Pull request by cmhhelgeson**
1. Fix overflow bug in panel element
2. Add brief descriptions of sample categories on hover

https://github.com/webgpu/webgpu-samples/assets/62450112/1f752779-0dc8-4a4a-a27d-d01fdd8b2891

### CODE REVIEW

**Review by kainino0x — APPROVED** [review]
FYI we are planning to do #355 in some way and I think @greggman has started looking at it. We might want to hold off on more changes after this PR until that's done.

**Comment by cmhhelgeson** [comment]
> FYI we are planning to do #355 in some way and I think @greggman has started looking at it. We might want to hold off on more changes after this PR until that's done.

More changes to this branch specifically, or more changes overall?

File: src/pages/samples/[slug].tsx [hunk 1]
```diff
@@ -94,28 +94,55 @@ const pages: PageComponentType = {
...
-  createPageCategory('Benchmarks', benchmarkPages),
+  createPageCategory(
+    'Basic Graphics',
+    graphicsBasicsPages,
+    'Basic rendering functionality implemented with the WebGPU API.'
+  ),
+  createPageCategory(
+    'WebGPU Features',
+    webGPUFeaturesPages,
+    'Demos of WebGPU-specific features or WebGPU-specific implementations of features.'
```
[the remark this answers is not in the document]
Comment (cmhhelgeson) [reply src/pages/samples/[slug].tsx:124]: Fixed

**Review by kainino0x — APPROVED** [review]

**Comment by kainino0x** [comment]
> How long do you think that will be?

@greggman would have to answer that

**Comment by greggman** [comment]
I am working on #355. Hopefully I'll have something to review today or tomorrow

**Pull request merged**